Formula/deno.rb

```diff
@@ -0,0 +1,56 @@
+class Deno < Formula
+  desc "Command-line JavaScript / TypeScript engine"
+  homepage "https://deno.land/"
+  url "https://github.com/denoland/deno.git",
+    :tag      => "v0.6.0",
+    :revision => "22feb74ba12215597416b5531f8a557302283e79"
+
+  depends_on "llvm" => :build
+  depends_on "ninja" => :build
+  depends_on "node" => :build
+  depends_on "python@2" => :build # GN require Python 2.7+
```

Comment:
Doh! I spoke too soon.

Please remove this and test with system python. I know at least mojave ships 2.7.x and I believe the other three OSes do too.

Reply:
LGTM! Thanks! I think we're there.